src/components/FlatParameters/FlatParameters.js

```diff
@@ -228,8 +233,8 @@ export const FlatParameters = ({
                         isNaN(fieldValue - 0));
                 return (
                     <TextField
-                        fullWidth
-                        sx={{ input: { textAlign: 'right' } }}
+                        size={'small'}
+                        sx={{ input: { textAlign: 'right' }, width: '50%' }}
```

Comment:
Maybe we should separate css rules in the inputProps prop of TextField from sx of the TextField component itself

Reply:
Done